### PR TITLE
Add 8.0.0 changelog

### PR DESCRIPTION
**Pull request by mmoll**
[ci-skip]

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## 8.0.0
+* New or changed parameters:
+    * Add autosign_content parameter to allow to supplying content for the
+      autosign file
+    * Add ca_crl_filepath, server_ca_crl_sync and server_crl_enable parameters.
+      This allows the CRL to be enabled when puppet_ca is disabled and provides
+      the ability to sync #{ssldir}/ca/ca_crl.pem contents to #{ssldir}/crl.pem
```
Comment (domcleal) [inline CHANGELOG.md]: Nitpick, use `$` for variables.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## 8.0.0
+* New or changed parameters:
+    * Add autosign_content parameter to allow to supplying content for the
```
Comment (domcleal) [inline CHANGELOG.md]: "parameter to supply content"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+      autosign file
+    * Add ca_crl_filepath, server_ca_crl_sync and server_crl_enable parameters.
+      This allows the CRL to be enabled when puppet_ca is disabled and provides
+      the ability to sync #{ssldir}/ca/ca_crl.pem contents to #{ssldir}/crl.pem
+      from a master of masters.
+    * Add server_ssl_key_manage paramter to enable external certificate and key
+      handling add a new boolean to disable the standard private_key.
```
Comment (domcleal) [inline CHANGELOG.md]: Unclear what the "add a new boolean" bit is referring to?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+    * Add server_ssl_chain_filepath parameter, to specify the value of
+      ssl-cert-chain in the webserver.conf file for puppetserver.
+    * Add server_allow_header_cert_info paramter to set allow-header-cert-info
```
Comment (domcleal) [inline CHANGELOG.md]: "parameter"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,38 @@
...
+      for puppetserver independently from the server_http parameter.
+* Other features:
+    * Support native puppetserver package on FreeBSD
+    * Allow disabling crl when server_ca => true
```
Comment (domcleal) [inline CHANGELOG.md]: Wrap `server_ca => true` in backticks.

**Comment by mmoll** [comment]
Should be all fixed, I also sprinkled some more backticks in...

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**